howso/direct/client.py

```diff
@@ -2376,6 +2378,11 @@ def _react_series(self, trainee_id, react_params):
         ret = dict()
 
         ret['action_features'] = batch_result.pop('action_features') or []
+        if 'influential_cases' in batch_result:
```

Comment:
We should format the response similar to how we do for normal reacts. See `format_react_response`. Also, instead of manually pulling out the details we should include all of them automatically.